website/docs/d/is_backup_policy.html.markdown

```diff
@@ -0,0 +1,58 @@
+---
+subcategory: "VPC infrastructure"
+layout: "ibm"
+page_title: "IBM : ibm_is_backup_policy"
+description: |-
+  Get information about backup policy.
+---
+
+# ibm\_is_backup_policy
```

Comment:
correction in the names here, please remove the \ for all the documentations.
Changes in all the documentation